bin/node/pallets/pallet-updater/src/functions.rs

```diff
@@ -0,0 +1,21 @@
+use super::*;
+use frame_support::dispatch::DispatchResult;
+
+impl<T: Config> Pallet<T> {
+	pub fn set_code(code: Vec<u8>) -> DispatchResult {
+        <Pallet<T>>::update_code_in_storage(&code)?;
+        Ok(())
+	}
+
+    pub fn can_set_code(code: &[u8]) -> Result<(), sp_runtime::DispatchError> {
+        // Check whether or not it is possible to update the code.
+
+        Ok(())
+    }
+
+    pub fn update_code_in_storage(code: &[u8]) -> DispatchResult {
+        // Write code to the storage and emit related events and digest items.
+        
+        Ok(())
+    }
+}
```

Comment:
We are probably going to use `frame/system`. Remove these and we'll introduce them later if needed.

Reply:
> We are probably going to use `frame/system`. Remove these and we'll introduce them later if needed.

Ok. Sounds good.